CONTRIBUTING.md

```diff
@@ -12,7 +12,19 @@ This guide will help you get started and outline some things you should know whe
 
 ## Setup
 
-* fork and clone the repository to your machine
+* install [Go](https://golang.org)
+  * on macOS via `brew install go`
+* set up the Go directory structure on your machine
+  * set the environment variable `$GOPATH` to your Go workspace
+    (you can point it to any folder on your hard drive, let's assume `~/go` here)
+  * add `~/go/bin` to your `$PATH`
+  * create the directory `~/go/src/github.com/Originate`
+  * cd into that directory, and run `git clone git@github.com:Originate/git-town.git gt`
+  * cd into `gt`
```

Comment:
Go uses the convention to use the folder name as the binary name, so we have to name the folder `gt` here, not `git-town`. Otherwise it would compile a binary `git-town`.